webpack/localization-plugin/README.md

```diff
@@ -122,6 +122,38 @@ translatedStrings: {
 }
 ```
 
+Alternatively, instead of directly specifying the translations, a path to a translated resource file can be
+specified. For example:
+
+```JavaScript
+translatedStrings: {
+  "en-us": {
+    "./src/strings1.loc.json": "./localization/en-us/strings1.loc.json"
```

Comment:
Document the rules on where `.` resolves to?

Reply:
It's in the README. It's relative to the webpack compilation context.